### DISCUSSION TITLE
Expandable Row Groups not working with second page onwards with PrimeNg-14.2.3 data table

### DISCUSSION THREAD

**Discussion by shanmuga-act** [primefaces/community#135] [PrimeNG] [answered]
Expandable Row Groups not working with second page onwards with PrimeNG-14.2.3 data table. But it's working fine with the first page of the data table. On second page, group header getting appended for every rows.
It was working well with PrimeNg-13.

![image](https://github.com/primefaces/community/assets/83413798/4be88e7b-0317-4913-996f-0a61e65041ca)

**Comment by gucal** [comment]
Hi,

Could you please create a Stackblitz example so we can identify the issue?

**Reply by shanmuga-act** [reply]
https://stackblitz.com/edit/stackblitz-starters-byt7mz?file=src%2Fmain.ts

**Reply by cetincakiroglu** [reply, marked as answer]
Hi, 

Thank you for sharing an example, it seems that a bug appears when the expandable row group feature is used with paginator. We've opened an issue for it. Here is the issue link: https://github.com/primefaces/primeng/issues/13783

Thanks for reporting the issue!

**Reply by shanmuga-act** [reply]
Any ETA on this?

**Reply by cetincakiroglu** [reply]
Right now we can't provide an ETA for that fix since we're fully focused on accessibility implementation to PrimeNG and the component code structure will change. Afterward, we'll take a look at it.
